keyboard.html

```diff
@@ -61,17 +63,17 @@
       document.getElementById("description").innerHTML = `Below should be a keyboard with ${totalKeyCount} keys.`;
     }
 
-    function addWhiteKey(keyCount) {
+    function addWhiteKey(keyCount, octaveCount) {
       const newDiv = document.createElement("div");
       newDiv.classList.add('white-key')
-      newDiv.setAttribute('key-name', `${keyNames[keyCount]}`)
+      newDiv.setAttribute('key-name', `${keyNames[keyCount]}${octaveCount}`)
```

Comment:
maybe another attribute for octive would make lookup easier and code cleaner
```suggestion
      newDiv.setAttribute('key-name', keyNames[keyCount])
      newDiv.setAttribute('octive', octaveCount)

```

Reply:
likely